<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="demo" time="1.079" tests="2" failures="0" errors="2">
   <testsuite name="demo" tests="2" failures="0" errors="2" time="1.079" skipped="0" timestamp="31-03-2025T11:30:50" hostname="**********" id="Test Suites/demo">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform"/>
         <property name="logFolder" value="/Users/gv.vignesh/Downloads/WGR-Android-Automation/Reports/20250331_113045/demo/20250331_113045"/>
         <property name="logFiles" value="/Users/gv.vignesh/Downloads/WGR-Android-Automation/Reports/20250331_113045/demo/20250331_113045/execution0.log"/>
         <property name="attachments" value=""/>
         <property name="hostName" value="**********"/>
         <property name="hostAddress" value="**********"/>
         <property name="os" value="Mac OS X 64bit"/>
         <property name="katalonVersion" value="10.1.1.0"/>
      </properties>
      <testcase name="Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage" time="0.031" classname="Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage" status="ERROR">
         <error type="ERROR" message="Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.&#xa;Reason:&#xa;java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'&#xa;	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)&#xa;	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)&#xa;	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)&#xa;	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)&#xa;	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)&#xa;	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)&#xa;	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)&#xa;	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)&#xa;	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)&#xa;	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xa;	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xa;	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xa;"/>
         <system-out><![CDATA[31-03-2025T11:30:51 - [TEST_CASE][ERROR] - Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage: Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)
	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)


31-03-2025T11:30:51 - [TEST_STEP][ERROR] - testData = findTestData("Data Files/LogiNewUsers"): Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)
	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)


31-03-2025T11:30:51 - [MESSAGE][ERROR] - Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)
	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)]]></system-out>
         <system-err><![CDATA[31-03-2025T11:30:51 - [TEST_CASE][ERROR] - Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage: Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)
	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)]]></system-err>
      </testcase>
      <testcase name="Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage" time="0.048" classname="Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage" status="ERROR">
         <error type="ERROR" message="Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage FAILED.&#xa;Reason:&#xa;java.lang.IllegalArgumentException: Invalid column name 'confirmpassword' for test data '/Users/gv.vignesh/Desktop/TestData.xlsx'. Possible values are '{email,firstname,lastname,password,confirm password}'&#xa;	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)&#xa;	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)&#xa;	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)&#xa;	at C698-C696_SignUpNewUserPage.run(C698-C696_SignUpNewUserPage:36)&#xa;	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)&#xa;	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)&#xa;	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)&#xa;	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)&#xa;	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)&#xa;	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)&#xa;	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)&#xa;	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)&#xa;	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#xa;	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#xa;	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#xa;"/>
         <system-out><![CDATA[31-03-2025T11:30:51 - [TEST_CASE][ERROR] - Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage: Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'confirmpassword' for test data '/Users/gv.vignesh/Desktop/TestData.xlsx'. Possible values are '{email,firstname,lastname,password,confirm password}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at C698-C696_SignUpNewUserPage.run(C698-C696_SignUpNewUserPage:36)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)


31-03-2025T11:30:51 - [TEST_STEP][PASSED] - testData = findTestData("Data Files/LogiNewUser"): null

31-03-2025T11:30:51 - [TEST_STEP][PASSED] - email = testData.getValue("email", 2): null

31-03-2025T11:30:51 - [TEST_STEP][PASSED] - firstName = testData.getValue("firstname", 2): null

31-03-2025T11:30:51 - [TEST_STEP][PASSED] - lastName = testData.getValue("lastname", 2): null

31-03-2025T11:30:51 - [TEST_STEP][PASSED] - password = testData.getValue("password", 2): null

31-03-2025T11:30:51 - [TEST_STEP][ERROR] - confirmPassword = testData.getValue("confirmpassword", 2): Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'confirmpassword' for test data '/Users/gv.vignesh/Desktop/TestData.xlsx'. Possible values are '{email,firstname,lastname,password,confirm password}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at C698-C696_SignUpNewUserPage.run(C698-C696_SignUpNewUserPage:36)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)


31-03-2025T11:30:51 - [MESSAGE][ERROR] - Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'confirmpassword' for test data '/Users/gv.vignesh/Desktop/TestData.xlsx'. Possible values are '{email,firstname,lastname,password,confirm password}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at C698-C696_SignUpNewUserPage.run(C698-C696_SignUpNewUserPage:36)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)]]></system-out>
         <system-err><![CDATA[31-03-2025T11:30:51 - [TEST_CASE][ERROR] - Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage: Test Cases/Sign Up New UserPage/C698-C696_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Invalid column name 'confirmpassword' for test data '/Users/gv.vignesh/Desktop/TestData.xlsx'. Possible values are '{email,firstname,lastname,password,confirm password}'
	at com.kms.katalon.core.testdata.AbstractTestData.verifyColumnName(AbstractTestData.java:78)
	at com.kms.katalon.core.testdata.AbstractTestData.getObjectValue(AbstractTestData.java:136)
	at com.kms.katalon.core.testdata.AbstractTestData.getValue(AbstractTestData.java:107)
	at C698-C696_SignUpNewUserPage.run(C698-C696_SignUpNewUserPage:36)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)]]></system-err>
      </testcase>
      <system-out><![CDATA[31-03-2025T11:30:50 - [TEST_SUITE][ERROR] - demo: Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)
	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)]]></system-out>
      <system-err><![CDATA[31-03-2025T11:30:50 - [TEST_SUITE][ERROR] - demo: Test Cases/Sign Up New UserPage/C699_SignUpNewUserPage FAILED.
Reason:
java.lang.IllegalArgumentException: Cannot find test data with id 'Data Files/LogiNewUsers' because (of) 'java.lang.IllegalArgumentException: Test data with id 'Data Files/LogiNewUsers' does not exist'
	at com.kms.katalon.core.testdata.TestDataFactory.findTestData(TestDataFactory.java:132)
	at C699_SignUpNewUserPage.run(C699_SignUpNewUserPage:22)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:448)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:439)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:418)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:410)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:285)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:71)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:160)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:107)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1743400845115.run(TempTestSuite1743400845115.groovy:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)]]></system-err>
   </testsuite>
</testsuites>
